.github/workflows/e2e-advanced-tests.yml

```
@@ -0,0 +1,123 @@
+          TURSO_DATABASE_URL: ${{ secrets.TURSO_DATABASE_URL }}
+          TURSO_AUTH_TOKEN: ${{ secrets.TURSO_AUTH_TOKEN }}
+          STRIPE_PUBLISHABLE_KEY: ${{ secrets.STRIPE_TEST_PUBLISHABLE_KEY }}
+          STRIPE_SECRET_KEY: ${{ secrets.STRIPE_TEST_SECRET_KEY }}
+          BREVO_API_KEY: ${{ secrets.BREVO_TEST_API_KEY }}
+          BREVO_NEWSLETTER_LIST_ID: ${{ secrets.BREVO_TEST_NEWSLETTER_LIST_ID }}
+          ADMIN_SECRET: ${{ secrets.TEST_ADMIN_SECRET }}
+          ADMIN_PASSWORD: ${{ secrets.TEST_ADMIN_PASSWORD }}
+          MIGRATION_SECRET_KEY: ${{ secrets.MIGRATION_SECRET_KEY }}
+          WALLET_AUTH_SECRET: ${{ secrets.WALLET_AUTH_SECRET }}
+          E2E_TEST_MODE: true
+          DATABASE_URL: "file:./data/e2e-test.db"        run: |
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 57: mapping values are not allowed in this context

(syntax-check)